### PR TITLE
Env template

### LINKED ISSUE #288: Create .env.template file
## Description
As a developer,
I need `.env.template` file,
so that I can minimize work while setting up the project environment variables.

The idea is to create a `.env.template` file to make developer's life easier while setting up the environment variables of the project, by having a template ready to use and place the variables.

### Update [Required]
- [ ] Create `.env.template` file with placeholders for environment variables
- [ ] Update README to indicate that the template exists to be used

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation

1 hour

### PR DESCRIPTION
**Pull request by PratibhaShrivastav**
### Description
Created the .env template file with the required environment variables and updated the readme accordingly.

Fixes #288 

### Type of Change:

- Documentation

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas

### CODE REVIEW

File: .env.template
Comment (isabelcosta) [inline .env.template]: Can you please add a blank line at the end of this file?

File: README.md
Comment (isabelcosta) [inline README.md:31]: Can you leave part as it is? so that people at least know where they can find more information about these environment variables. you can just leave the 2 options on step 4.

**Review by isabelcosta — CHANGES_REQUESTED** [review]
@PratibhaShrivastav thank you so much for your contribution!
I requested some changes on this PR, could you fix the PR accordingly?

**Comment by PratibhaShrivastav** [comment]
@isabelcosta I have done the required changes. Please review.

**Review by isabelcosta — APPROVED** [review]
Well done! @PratibhaShrivastav 
I will merge this, once we start merging work from Google Code-In :)

**Pull request merged**
